frontend/src/features/comparisons/CriteriaSlider.tsx

```diff
@@ -52,6 +56,20 @@ const CriteriaSlider = ({
   const { t } = useTranslation();
   const classes = useStyles();
 
+  const computeMedian = function computeMedian(
```

Comment:
(very minor)
unnecessary to repeat the function name:
```suggestion
  const computeMedian = function (
```
```